src/services/slack.ts

```diff
@@ -22,8 +23,8 @@ function timeSafeCompare(a: string, b: string) {
 
 export interface ISlackConnector {
   validateSlackRequest(payload: any): boolean;
-  displayRepoOptions(repos: Array<string>, triggerId: string): Promise<any>;
-  parseSelection(payload: any): any;
+  displayRepoOptions(repos: Array<string>, triggerId: string, admin: boolean): Promise<any>;
+  parseSelection(payload: any, isAdmin: boolean, repoBranchesRepository: RepoBranchesRepository): any;
```

Comment:
nit: could we name these parameters the same? I like the `isAdmin` to make it obviously a boolean